DParser2/Resolver/DTypeToCodeVisitor.cs

```diff
@@ -98,7 +100,29 @@ public void VisitDelegateType(DelegateType t)
 		void VisitDSymbol(DSymbol t)
 		{
 			var def = t.Definition;
-			sb.Append(def != null ? def.ToString(false, true) : "<Node object no longer exists>");
+			if (def == null)
+				sb.Append("<Node object no longer exists>");
+			else if (typeInstance)

```

Comment:
Hey, danke für den PR! :)
Sollte das Bauen des Strings nicht Teil von DSymbol selbst sein? Denn sonst wird ja nur `def.ToString()` aufgerufen, worin bis jetzt ja die ganze Logik lag.
Dann wären z.B. Debug-Ausgaben von DSymbol-Objekten auch sinnvoller/korrekter.

Reply:
Du meinst in `DSymbol.ToString`? Da wird ja z.B. in StructType `(struct)` vorangestellt, und das kann man hier nicht gebrauchen. Und `ToCode` landet ja hier. Vielleicht darf aber typeInstance immer true sein?

Reply:
Oh, hatte das mit dem ToCode nicht mehr in Erinnerung. Aber ja, dann lieber das Flag entfernen und immer die templateinstance parameter anzeigen, und nicht noch einen Zustand einbauen :)

Reply:
Ok, entfernt. Tests laufen noch unverändert durch.